python/hail/table.py

```diff
@@ -105,6 +105,19 @@ def global_schema(self):
 
     @property
     def key(self):
+        """Return the row key schema.
+
+        Examples
+        --------
+
+        Get the row key names of the table
```

Comment:
end with colon?

Reply:
can we also write examples like these for matrix table? feel free to push back if you don't want to do it here.